app/controllers/test/components/mentoring/mentor_conversations_lists_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Test::Components::Mentoring::MentorConversationsListsController < ApplicationController
+  def show; end
```

Comment:
@kntsoriano Two questions:
1. Do you like this style or would you prefer it over two lines? 
2. Do we treat empty methods in controllers as a smell? (ie they should be deleted or implemented)?

Reply:
1. Prefer it like this.
2. Not really. I like having empty controllers on a method in order to have visibility on what actions are implemented.

Reply:
OK, cool. Can you add to the standards document for controllers pls.